express-mongodb/signup-app.js

```diff
@@ -0,0 +1,71 @@
+var express = require("express")
+var bodyParser = require("body-parser")
+var mongoose = require("mongoose")
+
+const app = express()
+
+app.use(bodyParser.json())
+app.use(express.static('../HTML'))
+
+app.use(bodyParser.urlencoded({
+    extended:true
+}))
+
+const mongodb = require('mongodb')
+const MongoClient = mongodb.MongoClient
+
+// Connection URI
+const url = 'mongodb+srv://gamequiz:gameQuizTeam@gamequiz.9o3my64.mongodb.net/test';
```

Comment:
do not hard-coded username and password in the code. please use .env file and store them in this file.

Reply:
Please use dotenv module.